.github/workflows/cypress.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         run: |
           sha=${{ github.sha }}
           sha8=${sha:0:8}
-          echo ::set-output name=workspace::e2e$sha8
+          echo workspace=e2e$sha8 >> $GITHUB_OUTPUT
```

Comment:
Just to make sure I understand: this line plus the dependency upgrade were the only actual changes here, right?

I was a bit confused with the changes within the `dist` folder, but I imagine that those files are auto generated? Do they really need to be committed? I don't have enough context on the development of github workflows, so sorry if they're amateur questions.

Reply:
no problem, Maíra! thanks for asking and revewing

yeah, as far as I understand these are the only changes needed to move on from the deprecated feature. The changes to the dist folder is due to how github actions need to be "deployed". Since the code is in typescript we need to compile it somehow before "publishing", and since github actions only run the code that is in the repository we must also commit them :-/

I don't really like this approach very much – keeping the compiled code in the repo – and there are some actions that do this build step for you, and only push them in a tag, but I didn't set this up for this repo yet

Reply:
Makes total sense, thanks for explaining! Just wanted to understand how it works so we can better help with reviews in the future. Thanks a lot for setting this repo in the first place ✨ .